integration/test/ParseRelationTest.js

```diff
@@ -43,6 +43,34 @@ describe('Parse Relation', () => {
       });
   });
 
+  it('can do consecutive adds (#2056)', done => {
+    const ChildObject = Parse.Object.extend('ChildObject');
+    const childObjects = [];
+    for (let i = 0; i < 2; i++) {
+      childObjects.push(new ChildObject({ x: i }));
+    }
+    const parent = new Parse.Object('ParentObject');
+    parent.save().then(parentAgain => {
```

Comment:
Could you please convert this to `await` syntax instead of promise chaining, which we are not using anymore.

Reply:
Fixed. Note that there are existing tests that make use of the old syntax.